nixos/modules/programs/sway-beta.nix

```diff
@@ -20,13 +29,27 @@ in {
       '';
     };
 
+    extraSessionCommands = mkOption {
+      type = types.lines;
+      default = "";
+      example = ''
+        export SDL_VIDEODRIVER=wayland
+        # needs qt5.qtwayland in systemPackages
+        export QT_QPA_PLATFORM=wayland
+        export QT_WAYLAND_DISABLE_WINDOWDECORATION="1"
```

Comment:
Needed to prepare environment. xkb options moved to sway config in sway-beta, so add another example here. Also mentioned in official wiki https://github.com/swaywm/sway/wiki#disabling-client-side-qt-decorations

Reply:
> xkb options moved to sway config in sway-beta, so add another example here

what do you mean "so add another example here" since it's no longer in the env vars?

Reply:
@colemickens I mean instead of example provided in main module https://github.com/NixOS/nixpkgs/blob/master/nixos/modules/programs/sway.nix#L33